target/linux/qualcommax/image/ipq60xx.mk

```diff
@@ -11,6 +11,20 @@ define Device/8devices_mango-dvk
 endef
 TARGET_DEVICES += 8devices_mango-dvk
 
+define Device/cambiumnetworks_xe3-4
+       $(call Device/FitImage)
+       $(call Device/UbiFit)
+       DEVICE_VENDOR := Cambium Networks
+       DEVICE_MODEL := XE3-4
+       BLOCKSIZE := 128k
+       PAGESIZE := 2048
+       DEVICE_DTS_CONFIG := config@cp01-c3-xv3-4
+       SOC := ipq6010
+       DEVICE_PACKAGES := -kmod-usb3 -kmod-usb-dwc3 -kmod-usb-dwc3-qcom kmod-usb2 \
```

Comment:
There isn't really a reason for dropping the USB kmods.

BTW, is the USB2.0 port even enabled as I dont see it being described?

Reply:
It's enabled from the imported file as I mentioned earlier.

```
#include "ipq6018-cp01-c1.dts"
...
&qusb_phy_1 {
	status = "okay";
};

&usb2 {
	status = "okay";
};
```